.circleci/continue_config.yml

```
@@ -0,0 +1,284 @@
+          name: Install Kedro plugins dependencies
+          command: |
+            conda activate kedro_plugins
+            cd <<parameters.plugin>>
+            python -m pip install -U pip setuptools wheel
+            pip install git+https://github.com/kedro-org/kedro@main
+            pip install -r test_requirements.txt -U
+
+  win_tests:
+    description: Run build on Windows
+    parameters:
+      python_version:
+        type: string
+      plugin:
+        type: string
+    steps:
+      - checkout
+      - win_setup_conda:
+          python_version: <<parameters.python_version>>
+      - win_setup_requirements:
+          plugin: <<parameters.plugin>>
+      - run:
+          # e2e tests are not currently runnable on CircleCI on Windows as
+          # those require the ability to run Linux containers:
+          # "The Windows executor currently only supports Windows containers.
+          # Running Linux containers on Windows is not possible for now"
+          # (from https://circleci.com/docs/2.0/hello-world-windows/)
+          name: Run unit tests
+          command: |
+            conda activate kedro_plugins
+            pytest <<parameters.plugin>>\tests
+
+jobs:
+  unit_tests:
+    parameters:
+      python_version:
+        type: string
+      plugin:
+        type: string
+    machine:
+      # Don't use 2018 image: https://discuss.circleci.com/t/24639/18
+      image: circleci/classic:201711-01
+      docker_layer_caching: true
+    environment:
+      python_version: <<parameters.python_version>>
+    steps:
+      - setup:
+          python_version: <<parameters.python_version>>
+          plugin: <<parameters.plugin>>
+      - run:
+          name: Run tests
+          command: |
+            make plugin=<<parameters.plugin>> test
+
+  e2e_tests:
+    parameters:
+      python_version:
+        type: string
+      plugin:
+        type: string
+    machine:
+      # Don't use 2018 image: https://discuss.circleci.com/t/24639/18
+      image: circleci/classic:201711-01
+      docker_layer_caching: true
+    environment:
+      python_version: <<parameters.python_version>>
+    steps:
+      - setup:
+          python_version: <<parameters.python_version>>
+          plugin: <<parameters.plugin>>
+      - run:
+          name: Run all end to end tests
+          command: |
+            make plugin=<<parameters.plugin>> e2e-tests
+
+  lint:
+    parameters:
+      python_version:
+        type: string
+      plugin:
+        type: string
+    machine:
+      # Don't use 2018 image: https://discuss.circleci.com/t/24639/18
+      image: circleci/classic:201711-01
+      docker_layer_caching: true
+    environment:
+      python_version: <<parameters.python_version>>
+    steps:
+      - setup:
+          python_version: <<parameters.python_version>>
+          plugin: <<parameters.plugin>>
+      - run:
+          name: Run pylint and flake8
+          command: |
+            make plugin=<<parameters.plugin>> lint
```

Since we only ever use `python_version = 3.8` here, I'd actually just hard code it in rather than parametrising it. Up to you though.

```suggestion
  lint:
    parameters:
      plugin:
        type: string
    machine:
      # Don't use 2018 image: https://discuss.circleci.com/t/24639/18
      image: circleci/classic:201711-01
      docker_layer_caching: true
    environment:
      python_version: 3.8
    steps:
      - setup:
          python_version: 3.8
          plugin: <<parameters.plugin>>
      - run:
          name: Run pylint and flake8
          command: |
            make plugin=<<parameters.plugin>> lint
```